### PR TITLE
added spec for assessment controller

### PR DESCRIPTION
**Pull request by Devyani606**

### CODE REVIEW

File: spec/acceptance/api/v1/admin/assessment_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  post '/api/v1/admin/assessments/generate' do
+    context 'with valid params and heders' do
+      parameter :organization_id, 'organization_id'
+      let!(:organization_id) { organization.id }
+      example 'create Assessments from admin panel' do
+        do_request
+        data = JSON.parse(response_body)
+        expect(status).to eq(200)
+        expect(data['message']).to eq('Assessment created successfully.')
```
Comment (somnath77) [inline spec/acceptance/api/v1/admin/assessment_controller_spec.rb:43]: You can test if assessment count has increased or not ?

File: spec/acceptance/api/v1/admin/assessment_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    context 'with valid params and heders' do
+      parameter :id, 'id'
+      parameter :organization_id, 'organization_id'
+      let!(:id) { assessment.id }
+      let!(:organization_id) { organization.id }
+      example 'update Assessments from admin panel' do
+        do_request
+        data = JSON.parse(response_body)
+        expect(status).to eq(200)
+        expect(data['message']).to eq('Assessment updated successfully')
```
Comment (somnath77) [inline spec/acceptance/api/v1/admin/assessment_controller_spec.rb:58]: check if attributes values gets changed or not

File: spec/acceptance/api/v1/admin/assessment_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+  delete '/api/v1/admin/assessments/:id' do
+    context 'with valid params and heders' do
+      parameter :id, 'id'
+      let!(:id) { assessment.id }
+      example 'delete Assessments from admin panel' do
+        do_request
+        data = JSON.parse(response_body)
+        expect(status).to eq(200)
+        expect(data['message']).to eq('Assessment deleted successfully.')
```
Comment (somnath77) [inline spec/acceptance/api/v1/admin/assessment_controller_spec.rb:71]: same as above, check if assessments count gets decreased

File: spec/acceptance/api/v1/admin/assessment_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      let!(:id) { assessment.id }
+      example 'get all results of candidate who invited for Assessments' do
+        do_request
+        data = JSON.parse(response_body)
+        expect(status).to eq(200)
+        expect(data['message']).to eq('success')
+      end
+    end
+  end
+end
```
Comment (somnath77) [inline spec/acceptance/api/v1/admin/assessment_controller_spec.rb:101]: Tests for show method is missing

File: spec/acceptance/api/v1/admin/assessment_controller_spec.rb (hunk 1 above)
Comment (Devyani606) [reply spec/acceptance/api/v1/admin/assessment_controller_spec.rb:43]: done

File: spec/acceptance/api/v1/admin/assessment_controller_spec.rb (hunk 2 above)
Comment (Devyani606) [reply spec/acceptance/api/v1/admin/assessment_controller_spec.rb:58]: done

File: spec/acceptance/api/v1/admin/assessment_controller_spec.rb (hunk 3 above)
Comment (Devyani606) [reply spec/acceptance/api/v1/admin/assessment_controller_spec.rb:71]: done

File: spec/acceptance/api/v1/admin/assessment_controller_spec.rb (hunk 4 above)
Comment (Devyani606) [reply spec/acceptance/api/v1/admin/assessment_controller_spec.rb:101]: alreday written

**Review by somnath77 — APPROVED** [review]

**Pull request merged**